examples/GetPowerBIActivityEvents.ps1

```diff
@@ -0,0 +1,95 @@
+﻿<#
+
+.Synopsis
+    Retrieves the audit activity events for a Power BI tenant. 
+.Description
+    Retrieves the audit activity events for a Power BI tenant with in a time window.
+    This script uses the MicrosoftPowerBIMgmt.Profile PowerShell module. if this module isn't installed it will be installed into your current user profile.
+.Parameter StartDateTime
+    Specifies the start of a timespan to retrieve audit activity events. It should be in UTC format and ISO 8601 compliant.
+    Both StartDateTime and EndDateTime should be within the same UTC day.
+.Parameter EndDateTime
+    Specifies the end of a timespan to retrieve audit activity events. It should be in UTC format and ISO 8601 compliant.
+    Both StartDateTime and EndDateTime should be within the same UTC day.
+.Parameter ActivityType
+    Filters the activity records based on this activity type.
+.Parameter User
+    Filters the activity records based on this user email.
+.Example
+    PS C:\> .\GetPowerBIActivityEvents.ps1 -StartDateTime 2019-09-12T00:00:00 -EndDateTime 2019-09-12T10:00:00 -ActivityType viewreport -User admin@contoso.com
```

Comment:
Is this script still useful? If not we can remove in a separate PR

Reply:
yeah. I dont this its necessary from a usage stand point. Sure I will remove it in a seperate PR.